application/src/test/java/org/thingsboard/server/controller/RpcControllerTest.java

```diff
@@ -40,7 +40,7 @@
 
 @DaoSqlTest
 @TestPropertySource(properties = {
-        "transport.http.max_payload_size=10000"
+        "server.http.max_payload_size=/api/image*/**=10;/api/**=10000"
```

Comment:
What is image endpoint for?

Reply:
just to check request is being processed by second pattern, not first